test/lifecycle_module_test.dart

```diff
@@ -280,6 +282,8 @@ void expectInLifecycleState(LifecycleModule module, LifecycleState state) {
 }
 
 Future<Null> gotoState(LifecycleModule module, LifecycleState state) async {
+  // wait for next event loop. fixes sync-async in Dart 2
+  await new Future.value(null);
```

Comment:
@Workiva/app-frameworks want to take a look at these closely to determine if we're going to have a sync-async problem with lifecycle module or if it's just the tests and mocks?

Reply:
well, goto state is only used internally in tests, so I'm not too concerned about this one.